src/dataset.js

```diff
@@ -259,15 +259,20 @@ export class Dataset {
      * ```
      * {
      *   "id": "WkzbQMuFYuamGv3YF",
-     *   "name": "d7b9MDYsbtX5L7XAj",
+     *   "name": "my-dataset",
      *   "userId": "wRsJZtadYvn4mBZmm",
-     *   "createdAt": "2015-12-12T07:34:14.202Z",
-     *   "modifiedAt": "2015-12-13T08:36:13.202Z",
-     *   "accessedAt": "2015-12-14T08:36:13.202Z",
-     *   "itemCount": 0
+     *   "createdAt": new Date("2015-12-12T07:34:14.202Z"),
+     *   "modifiedAt": new Date("2015-12-13T08:36:13.202Z"),
+     *   "accessedAt": new Date("2015-12-14T08:36:13.202Z"),
+     *   "itemCount": 14,
+     *   "cleanItemCount": 10
      * }
      * ```
      *
+     * @param {Object} [options={}]
+     *   Additional options passed to Apify API Client's
+     *   [getDataset](https://www.apify.com/docs/api/apify-client-js/latest#ApifyClient-datasets-getDataset) function.
+     *
      * @returns {Promise<Object>}
      */
     getInfo(options = {}) {
```

Comment:
As looking into client and API, I think that the options parameter is relic here as no options are needed when `datasetId` is provided. It's needed for token in `datasets.getDataset()` only when `username-datasetname` is used instead of ID.

Reply:
Good point, removing it.